dev/io.openliberty.grpc.1.0.internal.client/bnd.bnd

```diff
@@ -86,7 +87,7 @@ instrument.disabled: true
   io.netty:netty-handler-proxy;version=${nettyVersion},\
   io.netty:netty-resolver;version=${nettyVersion},\
   io.netty:netty-transport;version=${nettyVersion},\
-  io.openliberty.grpc.client.1.0.thirdparty;version=latest,\
+  io.grpc:grpc-netty;version=${grpcVersion},\
```

Comment:
Well the feature no longer is depending on the thirdparty bundle to get the classes at runtime. I thought I ran into build issues without this change, but I would have to try again to be sure.  I changed to this to align with the other functions to pull from the actual dependency instead of the generated build artifact.

Reply:
OK